app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  validates :user, presence: true
```

Comment:
Validations for user and post are not necessary as the associations below take care of those for you.